Autoprefixer.py

```diff
@@ -48,8 +48,24 @@ def has_selection(self):
 				return True
 		return False
 
-	def get_setting(self, key):
-		settings = self.view.settings().get('Autoprefixer')
+	@staticmethod
+	def get_setting(view, key):
+		settings = view.settings().get('Autoprefixer')
 		if settings is None:
 			settings = sublime.load_settings('Autoprefixer.sublime-settings')
 		return settings.get(key)
+
+class AutoprefixerPreSaveCommand(sublime_plugin.EventListener):
+	def on_pre_save(self, view):
+
+		if AutoprefixerCommand.get_setting(view, 'prefixOnSave') is True:
+
+			if int(sublime.version()) >= 3080:
+				self.sublime_vars = view.window().extract_variables()
+			else:
+				self.sublime_vars = {
+					'file_extension': splitext(view.file_name())[1][1:]
+				}
+
+			if self.sublime_vars['file_extension'] in ('css'):
```

Comment:
I would prefer to check the syntax instead. That way it would work on unsaved buffers with set syntax. I often do that.

Something like:

```python
view.settings().get('syntax') == 'Packages/CSS/CSS.tmLanguage'
```